applications/cdr/src/csv_util.erl

```diff
@@ -46,7 +46,7 @@ test_convert(AccountDb) ->
                                           [CdrDoc | Acc]
                                   end, [], JObjs),
             CsvData = json_objs_to_csv(CdrDocs),
-            maybe_save_csv(<<"test.csv">>, CsvData)
+            maybe_save_csv("test.csv", CsvData)
```

Comment:
This was the warning:
```
warn_failing_call csv_util.erl:66: The call filelib:ensure_dir(TestPath::binary()) breaks the contract (Name) -> 'ok' | {'error',Reason} when is_subtype(Name,filename() | dirname()), is_subtype(Reason,file:posix())
```

I have no idea why `binary()` is not accepted by Dialyzer here… I am still trying to have 17's Dialyzer to run against our code base (it gives much less false positives)